packages/main/src/db.ts

```diff
@@ -1,45 +1,48 @@
 import DB from "better-sqlite3-helper"
+import electronIsDev from "electron-is-dev"
 import * as fs from "fs"
 import * as path from "path"
 import xdgBasedir from "xdg-basedir"
 
-const db = process.env.NODE_ENV === "test" ? initTestDb() : initDb()
-db.pragma("foreign_keys = ON")
-
-if (process.env.NODE_ENV !== "test") {
-  fs.chmodSync(getDbPath(), 0o600)
-}
+const db =
+  process.env.NODE_ENV === "test" ? initTestDb() : initDb(electronIsDev)
 
 export default db
 
-function initDb() {
-  return DB({
-    path: getDbPath(),
+function initDb(isDev: boolean): ReturnType<typeof DB> {
+  const path = getDbPath("db.sqlite", isDev)
+  const instance = DB({
+    path,
     migrate: {
       migrationsPath: getMigrationsPath()
     },
     WAL: false
   })
+  instance.pragma("foreign_keys = ON")
+  fs.chmodSync(path, 0o600)
+  return instance
 }
 
 function initTestDb() {
-  return DB({
+  const instance = DB({
     memory: true,
     migrate: {
       migrationsPath: getMigrationsPath()
     },
     WAL: false
   })
+  instance.pragma("foreign_keys = ON")
+  return instance
 }
 
-function getDbPath() {
+export function getDbPath(dbName: string, isDev: boolean): string {
   const cacheDir = xdgBasedir.cache
   if (!cacheDir) {
     throw new Error(
       "Could not locate XDG cache location! Are you running without a home directory?"
     )
   }
-  return path.join(cacheDir, "poodle", "db.sqlite")
+  return path.join(cacheDir, isDev ? "poodle-dev" : "poodle", dbName)
```

Comment:
I made a change so that the app uses a different cache location for development mode vs production. That will allow us to use the installed version of Poodle for day-to-day email tasks, while developing and testing the app without interfering with the production databases.